internal/clients/sources/stubs/sources_stub.go

```diff
@@ -26,6 +32,46 @@ func (m *contextReadError) Error() string {
 	return "failed to find or convert dao stored in testing context"
 }
 
+// SourcesClientV2
+func WithSourcesClientV2(parent context.Context, init_store *[]sources.Source) context.Context {
+	ctx := context.WithValue(parent, sourcesCtxKey, &SourcesClientV2Stub{store: init_store})
```

Comment:
I do not see the store being used anywhere, imho we can just get rid of it (no store) and keep just the static data.
Our code should not change data of sources, so I don't see a point in passing it dynamically.